config/testdata/force/source.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: source.toolkit.fluxcd.io/v1beta1
+kind: GitRepository
+metadata:
+  name: force
+spec:
+  interval: 10m
+  url: https://gist.github.com/cd83f83eebee3217d02ee5d3b97a9908.git
```

Comment:
Probably need a better place to keep these.

Reply:
Please remove these from e2e, we should add unit tests using an immutable fields.